### PR TITLE
feat: us0003

### LINKED ISSUE #9: US - 0003
Criação da rota com listagem de todos os usuários que seguem o vendedor e implementação do serviço.

### PR DESCRIPTION
**Pull request by Victor-Planas**
feat: us0003 - created user controller route and implemented service

### CODE REVIEW

**Review by lmarizmeli — APPROVED** [review]
🐿️

File: src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/controllers/UsersController.java [hunk 1]
```diff
@@ -61,4 +62,11 @@ public GenericMessageDTO followSeller(@PathVariable @Min(0) Integer userId, @Pat
         usersService.follow(userId,userIdToFollow);
         return new GenericMessageDTO("Seller followed succesfully!");
     }
+
+    @GetMapping("/{userId}/followers/list")
+    @ResponseStatus(HttpStatus.OK)
+    @Operation(description = "List all customers that follow given seller")
```
Comment (CarlosFeldmann) [inline src/main/java/br/com/mercadolivre/bootcampw2/grupo11/socialmeli/controllers/UsersController.java]: Follow given ?

**Review by CarlosFeldmann — CHANGES_REQUESTED** [review]

**Review by CarlosFeldmann — APPROVED** [review]
Lindo 🦧🦧🦧🦧🦧

**Review by Giovannapls — APPROVED** [review]
Quente 🔥☄️

**Pull request merged**